docs/en/development/build.md

```diff
@@ -85,6 +85,10 @@ git clone --recursive --shallow-submodules https://github.com/ClickHouse/ClickHo
 
 ### Build ClickHouse {#build-clickhouse}
 
+:::tip
+By default, `cmake` will use only one thread, while `ninja` will automatically detect and use the appropriate number of threads for the current system. You can tweak `cmake` to use a specific number of threads with `-j` flag, for example, `cmake --build build -j 16`. Alternatively, you can generate build files with a specific number of jobs in advance to avoid setting the flag all the time: `cmake -DPARALLEL_COMPILE_JOBS=16 -S . -B build`, where `16` is the desired number of threads.
```

Comment:
@rschu1ze is it a known issue? On Ubuntu, it works w/o any "sit-ups" for me

Reply:
It probably depends on the Linux/cmake distro? On Fedora 38, it is only one thread by default.

Reply:
I'd rather paraphrase it like, "In case `cmake` wasn't able to detect the number of available logical cores, the build will be done by one thread. To overcome it ..."

Reply:
Updated.